src/components/molecules/PollBox/PollBox.scss

```
@@ -0,0 +1,54 @@
+@import "scss/constants";
+
+.PollBox {
+  $spacing: 10px;
```

We tend to define our variables outside of the classes.

```suggestion

$spacing: 10px;

.PollBox {
```